cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "testdeploy",
+	Usage:    "deploys test ganache",
+	Category: "TESTING",
+	Flags:    TestFlags,
+	Description: "\tThe testdeploy command is used to deploy a ganache instance for testing\n" +
+		"\tTo add additional accounts, -testaccount\n" +
+		"\tTo only use custom accounts, -reset\n" +
+		"\tTo use a mnemonic, -mnemonic",
+}
+
+// The Private Keys of Alice, Bob, Etc
+var BaseAccounts = []string{"0x000000000000000000000000000000000000000000000000000000416c696365", "0x0000000000000000000000000000000000000000000000000000000000426f62", "0x00000000000000000000000000000000000000000000000000436861726c6965", "0x0000000000000000000000000000000000000000000000000000000044617665", "0x0000000000000000000000000000000000000000000000000000000000457665"}
+
+// Args holds the args to run tests with ganache
+type Args struct {
+	port     string
+	amount   string
+	mnemonic string
+	accounts []string
+}
+
+func newArgs() *Args {
+	return &Args{
+		accounts: BaseAccounts,
+		port:     "8545",
+		amount:   "100000000000000000000",
+	}
+}
+
+func handleDeployCmd(ctx *cli.Context) error {
+	log.Info("Starting Ganache")
+	gopath := os.Getenv("GOPATH")
+
+	args := newArgs()
+
+	if newPort := ctx.String(PortFlag.Name); newPort != "" {
+		args.port = newPort
+	}
```

Comment:
For future reference, you can save yourself some logic by using the `flag.Value` field to set defaults. Then you can just do `args.port = ctx.String(PortFlag.Name)`